fandogh_cli/service_commands.py

```diff
@@ -196,7 +196,27 @@ def service_apply(file, parameters):
         Managed service deployed successfully
         """
 
-    click.echo(message)
+    if detach:
+        click.echo(message)
+    else:
+        while True:
+            details = get_details(service_name)
+
+            if not details:
+                exit(1)
+
+            click.clear()
+
+            if details.get('state') == 'RUNNING':
+                present_service_detail(details)
+                click.echo(message)
+                exit(1)
+            elif details.get('state') == 'UNSTABLE':
+                present_service_detail(details)
+                click.echo('You can press ctrl + C to exit details service state monitoring')
+                sleep(3)
+            else:
+                exit(1)
```

Comment:
What does it mean when state is neither `RUNNING` nor `UNSTABLE`?

Reply:
it means nothing yet cause we have to solid document yet for states